src/components/pill-selector-dropdown/RoundPill.scss

```diff
@@ -67,6 +80,26 @@
     .bdl-Avatar-externalBadge {
         border-color: $bdl-gray-20;
     }
+
+    &.bdl-RoundPill--warning {
+        background-color: $bdl-yellorange-20;
+        border-color: $bdl-yellorange;
+        box-shadow: none;
```

Comment:
were these inheriting a box shadow from somewhere?

Reply:
Yes, the default selected pill style has a white box shadow. On pills with borders it looks like a gap though